cpp/tests/copying/nvstring_scatter_tests.cu

```diff
@@ -26,10 +26,9 @@
 #include <tests/utilities/nvcategory_utils.cuh>
 #include <tests/utilities/valid_vectors.h>
 
-template <typename T>
```

Comment:
@jrhemstad could use confirmation this was the correct thing to do here

Reply:
@harrism could use a quick set of eyes here

Reply:
There wasn't a `using...` clause so I didn't see any type parametrization which is why we removed this. We tried using `TYPED_TEST` without defining type parameters and it caused compiler errors.